projects/rulenode-core-config/src/lib/components/action/delete-attributes-config.component.ts

```diff
@@ -36,7 +37,14 @@ export class DeleteAttributesConfigComponent extends RuleNodeConfigurationCompon
     this.deleteAttributesConfigForm = this.fb.group({
       scope: [configuration ? configuration.scope : null, [Validators.required]],
       keys: [configuration ? configuration.keys : null, [Validators.required]],
-      sendAttributesDeletedNotification: [configuration ? configuration.sendAttributesDeletedNotification : false, []]
+      sendAttributesDeletedNotification: [configuration ? configuration.sendAttributesDeletedNotification : false, []],
+      notifyDevice: [configuration ? configuration.sendAttributesDeletedNotification : true, []]
```

Comment:
why configuration ? configuration.sendAttributesDeletedNotification?

Reply:
the default value should be false, we need set it to true by default only if scope is set to SHARED_SCOPE